python/py-authres/Portfile

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+set real_name       authres
+name                py-${real_name}
```

Comment:
There is no need to create a `real_name` variable, because the python portgroup gives you `${python.rootname}` once you set the name normally.

`name                py-authres
`